content/workload-reduction-toolkit/identify-workload-issues/have-structured-conversations-to-identify-issues.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Have structured conversations to identify issues
+colour: pink
+---
+
+# Have structured conversations to identify issues
+
+<strong class="govuk-tag">Template</strong>
+
+A structured conversation template to help gather information from staff and
+teachers about their workload issues.
+
+<div class="govuk-grid-row dfe-width-container govuk-!-padding-bottom-6">
+  <div class="govuk-grid-column-full">
+    <div class="info-box">
+      <div class="info-box__corner">
+        <img src="/assets/images/clipboard-icon.svg" alt="Clipboard icon">
```

Comment:
I think this should be the download icon rather than the clipboard, which is just for the survey.